juniper_official/interface/check-optical-temp-thresholds.rule

```diff
@@ -70,19 +70,19 @@ healthbot {
                 description "Sets health based on the optical temperature";
                 frequency 1.5offset;
                 /*
-                 * Sets color to red when optical-temperature is greater-than-or-equal-to high-alarm-threshold.
+                 * Sets color to green when optical-temperature is less than high-warning-threshold.
                  */				
-                term greater-than-alarm-threshold {
+                term normal {
                     when {
-                        greater-than-or-equal-to "$optical-temp" "$high-alarm-threshold";
-                    }
+                        less-than "$optical-temp" "$high-warning-threshold";
+                    }				
                     then {
                         status {
-                            color red;
-                            message "$interface-name,temperature $optical-temp exceeds default alarm threshold ($high-alarm-threshold)";
+                            color green;
```

Comment:
All yellow thresholds needs to rechecked where only green and red are changed

Reply:
The latest rule in jcloud has the updated changes, it was merged as part of another MR.